actors/miner/tests/util.rs

```diff
@@ -558,47 +562,14 @@ impl ActorHarness {
         first_for_miner: bool,
         base_fee: &TokenAmount,
     ) -> Result<Option<IpldBlock>, ActorError> {
-        if self.options.use_v2_pre_commit_and_replica_update {
```

Comment:
Remove this option from HarnessOptions. Can probably rename `pre_commit_sector_batch_v2` to drop the v2 discriminator too, or better just flatten it into `pre_commit_sector_batch_inner`.